compiler/src/iree/compiler/Dialect/HAL/IR/test/interface_ops.mlir

```diff
@@ -29,6 +30,23 @@ func.func @interface_io_subspan(%dim0: index, %dim2: index) {
 
 // -----
 
+// CSE-LABEL: @interface_subspan_cse
+func.func @interface_subspan_cse() {
+  %c0 = arith.constant 0 : index
+
+//   CSE: %[[BIND:.+]] = hal.interface.binding.subspan set(0) binding(2) type(storage_buffer) alignment(64) offset({{.+}}) : !flow.dispatch.tensor<readwrite:tensor<2x32x16384xf32>
+  %0 = hal.interface.binding.subspan set(0) binding(2) type(storage_buffer) alignment(64) offset(%c0) : !flow.dispatch.tensor<readwrite:tensor<2x32x16384xf32>>
+  %1 = hal.interface.binding.subspan set(0) binding(2) type(storage_buffer) alignment(64) offset(%c0) : !flow.dispatch.tensor<readwrite:tensor<2x32x16384xf32>>
+
+//   CSE-NEXT: %[[LOAD:.+]] = flow.dispatch.tensor.load %[[BIND]], offsets = [0, 0, 0], sizes = [2, 32, 16384], strides = [1, 1, 1] : !flow.dispatch.tensor<readwrite:tensor<2x32x16384xf32>> -> tensor<2x32x16384xf32>
```

Comment:
Nit: could drop the `, offsets = ..` part given that's not relevant to what we are checking here.